### PR TITLE
Paris - Thomas & Samuel

### PR DESCRIPTION
**Pull request by petitkriket**
@tomdreamer

### CODE REVIEW

File: starter_code/queries.md [hunk 1]
```diff
@@ -4,80 +4,99 @@
...
 
 ### 11. All the companies founded on the second semester of the year. Limit your search to 1000 companies.
 
-<!-- Your Code Goes Here -->
+S {founded_month: {$gte: 4, $lte:8 }}
 
 ### 12. All the companies that have been 'deadpooled' after the third year.
 
-<!-- Your Code Goes Here -->
+NOPE
```
Comment (ta-web-paris) [inline starter_code/queries.md]: haha!

File: starter_code/queries.md [hunk 2]
```diff
@@ -4,80 +4,99 @@
...
 ### 10. Retrieve the 10 companies with more employees, order by the `number of employees`
 
-<!-- Your Code Goes Here -->
+S {"number_of_employees" :-1}
+L : 10
 
 ### 11. All the companies founded on the second semester of the year. Limit your search to 1000 companies.
 
-<!-- Your Code Goes Here -->
+S {founded_month: {$gte: 4, $lte:8 }}
```
Comment (ta-web-paris) [inline starter_code/queries.md]: and your limit? :)

**Comment by ta-web-paris** [comment]
Well done!

File: starter_code/queries.md (hunk 2 above)
Comment (petitkriket) [reply starter_code/queries.md]: no limit ta vu mais discret :)

**Pull request closed without merging**